app/views/users/nbp_wallet/connect.html.slim

```diff
@@ -0,0 +1,24 @@
+.header.mb-5
+  .std-heading
+    i.fa-solid.fa-wallet
+    =< t('.header')
+
+.container
```

Comment:
I am actually not sure whether we need this `div` here. It mainly seems to add some margin on the left hand side, moving the QR code a bit to the right.

Reply:
That's true. I removed that :+1: